examples/breakout/host/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![allow(unused)]
+
+mod graphics;
+mod gui;
+mod roc;
+
+#[no_mangle]
+pub extern "C" fn main() {
+    // let bounds = roc::Bounds {
+    //     width: 1900.0,
+    //     height: 1000.0,
+    // };
+
+    // gui::run_event_loop("RocOut!", bounds).expect("Error running event loop");
+    println!("TODO FIX PLATFORM IMPLEMENTATION");
+    std::process::exit(0);
+}
```

Comment:
@Anton-4 -- do you think we should try and keep both breakout and the gui platform? They aren't in a good state currently. I have made changes so they at least compile and are usable (though breakout I have basically broken entirely) for CI purposes. What is our plan/thoughts longer term?

Reply:
Yeah, let's take them out, I don't think they've been a net positive overall.